src/Microsoft.DotNet.XHarness.iOS/AppRunner.cs

```diff
@@ -347,7 +353,8 @@ public class AppRunner
                 }
             }
 
-            listener.Cancel();
+            // TODO: https://github.com/dotnet/xharness/issues/73
+            // listener.Cancel();
```

Comment:
Hmm that is annoying. Crazy question, have you tried something int the listener.Cancel to instead of calling the Cancel of the TcpListener to access the Socket and close it? Something like:
```csharp
server.Server.Close();
```
Is a little brute but might work.

Reply:
Thanks, will give it a go